ksql-engine/src/main/java/io/confluent/ksql/internal/KsqlEngineMetrics.java

```diff
@@ -31,10 +35,13 @@
 import org.apache.kafka.common.metrics.stats.Max;
 import org.apache.kafka.common.metrics.stats.Min;
 import org.apache.kafka.common.metrics.stats.Value;
+import org.apache.kafka.streams.KafkaStreams;
+import org.apache.kafka.streams.KafkaStreams.State;
 
 public class KsqlEngineMetrics implements Closeable {
 
   private final List<Sensor> sensors;
+  private final List<CountMetric> countMetrics;
```

Comment:
nit: can you clean up `numActiveQueries` while you're in here - this is another unused var. 

Once done, `configureNumActiveQueries` can return void.

My challenge to you it to get the file so that your IDE does not report any warnings!

Reply:
Done! Challenge accepted :) No warning in my IDE!